waisntechtools/alerts/get_twilio_cred.py

```diff
@@ -0,0 +1,17 @@
+from django.conf import settings
```

Comment:
May want to call this file `twilio.py` since it is more generic and the name won't need to change if more twilio functions are added.